lib/common.js

```diff
@@ -430,7 +430,7 @@ exports.createTableForProperties = function(type, opts, properties) {
 
 exports.createTableForTeams = function(teams) {
   var table = new Table({
-    head: ['id', 'Name'],
+    head: ['ID', 'Name'],
```

Comment:
This may broke automation scripts we have. Can we have it reverted? Is that change needed?